updater/library/src/updater.rs

```diff
@@ -146,13 +146,11 @@ fn update_internal(config: &ResolvedConfig) -> anyhow::Result<UpdateStatus> {
     download_to_path(&patch.download_url, &download_path)?;
 
     // Inflate the patch from a diff if needed.
-    if patch.is_diff {
-        let base_path = PathBuf::from(&config.original_libapp_path);
-        let output_path = download_dir.join(format!("{}.full", patch.number.to_string()));
-        inflate(&download_path, &base_path, &output_path)?;
-        download_path = output_path;
-    }
-
+    let base_path = PathBuf::from(&config.original_libapp_path);
+    let output_path = download_dir.join(format!("{}.full", patch.number.to_string()));
+    inflate(&download_path, &base_path, &output_path)?;
+    download_path = output_path;
```

Comment:
This no longer needs to re-use the same variable name now that it's no longer optional.